packages/twenty-server/src/main.ts

```diff
@@ -64,7 +61,7 @@ const bootstrap = async () => {
   // Create the env-config.js of the front at runtime
   generateFrontConfig();
 
-  await app.listen(app.get(EnvironmentService).get('PORT'));
+  await app.listen(process.env.PORT ?? 3000);
```

Comment:
Default value a bit duplicate from the env service, especially for variable that would most likely be accessed only through process.env and never in the context of the envService/app like this one. But I think it doesn't hurt too much this way - it's the best options to keep things that way imo